test/commands/status.test.js

```diff
@@ -5,7 +5,10 @@ const sinon = require('sinon');
 const assume = require('assume');
 
 const mockConfig = {
-  host: 'warehouse.ai',
+  hosts: {
+    wrhs: 'warehouse.ai',
+    status: 'warehouse.ai'
```

Comment:
make these 2 hosts different and make sure the status request goes to the right host